contrib/get-win-dependencies

```diff
@@ -60,9 +71,10 @@ echo ===========================================================================
 echo
 echo "Setting up ANTLR 3.4..."
 echo
-MACHINE_TYPE=x86 ANTLR_CONFIGURE_ARGS=--host=$HOST contrib/get-antlr-3.4 | grep -v 'Now configure CVC4 with' | grep -v '\./configure --with-antlr-dir='
+MACHINE_TYPE="x86_64" ANTLR_CONFIGURE_ARGS="--host=$HOST" contrib/get-antlr-3.4 | grep -v 'Now configure CVC4 with' | grep -v '\./configure --with-antlr-dir='
 [ ${PIPESTATUS[0]} -eq 0 ] || reporterror
 echo
+exit 0
```

Comment:
Is this `exit 0` intentional? Everything after this line will be skipped.

Reply:
Yeah, I put it there intentionally to test something at some point :P

Good catch, removed it in new commit.